cluster/src/test/java/com/hartwig/pipeline/metadata/ResearchMetadataApiTest.java

```diff
@@ -151,12 +153,14 @@ public void publishesPipelineStagedEventOnCompletionSomaticFile() throws Excepti
 
         PipelineStaged result =
                 ObjectMappers.get().readValue(pubsubMessageArgumentCaptor.getValue().getData().toByteArray(), PipelineStaged.class);
-        assertThat(result.analysisMolecule()).isEqualTo(Molecule.DNA);
-        assertThat(result.runId()).isEmpty();
-        assertThat(result.setId()).isEqualTo(SET_ID);
-        assertThat(result.sample()).isEqualTo("tumor");
-        assertThat(result.version()).isEqualTo(Versions.pipelineMajorMinorVersion());
-        PipelineOutputBlob blobResult = result.blobs().get(0);
+        assertThat(result.pipeline().runId()).isEqualTo(1);
+        assertThat(result.pipeline().setId()).isEqualTo(SET_ID);
+        assertThat(result.pipeline().sample()).isEqualTo("tumor");
+        assertThat(result.pipeline().version()).isEqualTo(Versions.pipelineMajorMinorVersion());
+
+        Analysis analysis = result.pipeline().analyses().get(1);
```

Comment:
Think the testcase names need some attention so they align with what you're doing here (picking out one part of the single event that is published).